addons/web_tour/static/src/js/running_tour_action_helper.js

```diff
@@ -28,6 +28,9 @@ var RunningTourActionHelper = core.Class.extend({
     text: function (text, element) {
         this._text(this._get_action_values(element), text);
     },
+    remove_text(text, element) {
+        this._text(this._get_action_values(element), '\n');
+    },
```

Comment:
Not sure this is worth it, maybe just write "text \n" in your tour?

Reply:
If I want to be able to do that, I will have to replace the regex inside `_to_next_running_step`'s `exec` because:
```js
"text \n".match(/^([a-zA-Z0-9_]+) *(?:\(? *(.+?) *\)?)?$/) === null
```
While I only use `<command><space><some parameters>`, it feels like this is intended for parsing something more specific (typically: space being optional, explicitly stopping at end of line). If you can confirm the supported syntax is supposed to be something like "split both sides of the first space group" (or keep whole text if no space), I'll adapt it here instead.

Reply:
Ok, let's keep the new "remove_text" util then :+1: